extension/src/telemetry/constants.ts

```diff
@@ -28,7 +28,7 @@ export const EventName = Object.assign(
     EXTENSION_EXECUTION_DETAILS_CHANGED: 'extension.executionDetails.changed',
     EXTENSION_LOAD: 'extension.load',
 
-    SET_EXPERIMENTS_HEADER_DEPTH: 'extension.updateHeaderDepth',
+    SET_EXPERIMENTS_HEADER_DEPTH: 'extension.updateHeaderHeight',
```

Comment:
[Q] Can you set this from the command palette (should look at adding)? The event is coming from the experiments table so should be a `views.experimentTable.` type event.

Reply:
> [Q] Can you set this from the command palette (should look at adding)?

Are you asking if the "update header depth" feature can be fired via the command palette? If that is what you're asking, then currently no, but I believe it is doable if we wanted that! 

> The event is coming from the experiments table so should be a views.experimentTable. type event.

Understood!